packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -255,76 +253,16 @@ export async function createNewFluidFileFromSummary(
-
-        return PerformanceEvent.timedExecAsync(
-            logger,
-            { eventName: "createNewFile" },
-            async (event) => {
-                const snapshotBody = JSON.stringify(containerSnapshot);
-                let url: string;
-                let headers: { [index: string]: string; };
-                let addInBody = false;
-                const formBoundary = uuid();
-                let postBody = `--${formBoundary}\r\n`;
-                postBody += `Authorization: Bearer ${storageToken}\r\n`;
-                postBody += `X-HTTP-Method-Override: POST\r\n`;
-                postBody += `Content-Type: application/json\r\n`;
-                postBody += `_post: 1\r\n`;
-                postBody += `\r\n${snapshotBody}\r\n`;
-                postBody += `\r\n--${formBoundary}--`;
-
-                if (postBody.length <= maxUmpPostBodySize) {
-                    const urlObj = new URL(initialUrl);
-                    urlObj.searchParams.set("ump", "1");
-                    url = urlObj.href;
-                    headers = {
-                        "Content-Type": `multipart/form-data;boundary=${formBoundary}`,
-                    };
-                    addInBody = true;
-                } else {
-                    const parts = getUrlAndHeadersWithAuth(
-                        initialUrl, storageToken, forceAccessTokenViaAuthorizationHeader);
-                    url = parts.url;
-                    headers = {
-                        ...parts.headers,
-                        "Content-Type": "application/json",
-                    };
-                    postBody = snapshotBody;
-                }
-
-                const fetchResponse = await runWithRetry(
-                    async () => epochTracker.fetchAndParseAsJSON<ICreateFileResponse>(
-                        url,
-                        {
-                            body: postBody,
-                            headers,
-                            method: "POST",
-                        },
-                        "createFile",
-                        addInBody,
-                    ),
-                    "createFile",
-                    logger,
-                );
-
-                const content = fetchResponse.content;
-                if (!content || !content.itemId) {
-                    throw new NonRetryableError(
-                        "ODSP CreateFile call returned no item ID",
-                        DriverErrorType.incorrectServerResponse,
-                        { driverVersion });
-                }
-                event.end({
-                    headers: Object.keys(headers).length !== 0 ? true : undefined,
-                    attempts: options.refresh ? 2 : 1,
-                    ...fetchResponse.propsToLog,
-                });
-                return content;
-            },
-        );
-    });
+    return createNewFluidContainerCore(
+        containerSnapshot,
```

Comment:
Now this logic has been refactored and moved to createNewUtils.ts to be reused by createNewContainerOnExistingFile